### DISCUSSION TITLE
SigNoz @ Github Externship Winter Cohort

### DISCUSSION THREAD

**Discussion by pranay01** [SigNoz/signoz#488] [General]
We ❤️  open source

Please share any points you want to discuss about SigNoz @ Github externships here

**Comment by abhijeet-chatterjee** [comment]
@pranay01 Can you give a brief explanation about what does building workflows on SigNoz means ?

**Reply by pranay01** [reply]
hey @abhijeet007rocks8 Talked about it in my Github India AMA here - https://youtu.be/nr793tw4CnE?t=4478
